credential/store/src/main/java/org/wildfly/security/credential/store/impl/KeyStoreCredentialStore.java

```diff
@@ -74,6 +74,9 @@
 import javax.crypto.spec.IvParameterSpec;
 import javax.crypto.spec.SecretKeySpec;
 
+import net.i2p.crypto.eddsa.EdDSAPrivateKey;
```

Comment:
@darranl I was wondering what are your thoughts on using the `net.itp.crypto` library to support generating and importing EdDSA keys into credential stores?

Reply:
@ehsavoie I think we will not support importing and generating EdDSA keys in credential stores until JDK has added support for them. Because we are using the apache mina resource parser to parse the keys specified as strings, we have the option of including the net.i2p.crypto.eddsa dependency to allow parsing of EdDSA keys directly from an wildfly-config.xml file. It is possible to exclude that dependency (in WildFly-Core and Elytron) and not support EdDSA keys for SSH authentication at all (including when users only specify the location to EdDSA keys). I was wondering what you think would be the better approach for this?